main.go

```diff
@@ -52,6 +54,11 @@ func main() {
 	stepconf.Print(cfg)
```

Comment:
Why do we have this? 🤔 
cfg.OutputTool = "xcpretty"
https://github.com/bitrise-steplib/steps-xcode-build-for-test/pull/11/files#diff-7ddfb3e035b42cd70649cc33393fe32cR48

Reply:
No idea.